pkg/ui/src/views/cluster/containers/nodeGraphs/dashboards/replication.tsx

```diff
@@ -7,10 +7,32 @@ import { Metric, Axis, AxisUnits } from "src/views/shared/components/metricQuery
 import { GraphDashboardProps, nodeDisplayName, storeIDsForNode } from "./dashboardUtils";
 
 export default function (props: GraphDashboardProps) {
-  const { nodeIDs, nodesSummary, storeSources } = props;
+  const { nodeIDs, nodesSummary, storeSources, tooltipSelection } = props;
 
   return [
-    <LineGraph title="Ranges" sources={storeSources}>
+    <LineGraph
+      title="Ranges"
+      sources={storeSources}
+      tooltip={(
+        <div>
+          {`Count of ranges ${tooltipSelection} with the following properties:`}
```

Comment:
I didn't notice this before, but since all your tooltips except one drop into JSX, these can be written more simply -- you don't need to go into backtick strings at all.

Instead of

```
{`Count of ranges ${tooltipSelection} with the following properties:`}
```

you can just write

```
Count of ranges {tooltipSelection} with the following properties:
```

everywhere.

Reply:
Thanks, will update to use this syntax!